### ISSUE TITLE
Soft hyphen (00AD) is invisible

### ISSUE THREAD

**Issue by drs-p** [googlefonts/geologica#16]
The soft hyphen in the Geologica fonts is invisible and has width zero. That is a problem in a Tagged PDF as soft hyphens should be used for hyphenated words to improve accessibility and screen reading. See the discussion at this issue: https://github.com/latex3/tagging-project/issues/967, and also https://jkorpela.fi/shy.html#pdf for a discussion about the soft hyphen in general and its use in PDF specifically.

**Comment by drs-p** [comment]
On second thoughts, there is some discussion whether this is the right thing to do (see the ongoing thread at latex3/tagging-project#967). So I'll close this issue for now, sorry for the noise!
